addons/mail/static/src/models/autocomplete_input_view.js

```diff
@@ -17,7 +38,54 @@ registerModel({
                 return;
             }
         },
-         /**
+        /**
+         * @private
+         * @param {MouseEvent} ev
+         */
+        onKeydown(ev) {
+            if (ev.key === 'Escape') {
+                this.onBlur();
+            }
+        },
+        /**
+         * @private
+         * @param {MouseEvent} ev
+         */
+        onSelect(ev, ui) {
+            if (this.chatWindowOwnerAsNewMessage) {
+                this.chatWindowOwnerAsNewMessage.onAutocompleteSelect(ev, ui);
+                return;
+            }
+            if (this.discussSidebarCategoryOwnerAsAddingItem) {
+                this.discussSidebarCategoryOwnerAsAddingItem.onAddItemAutocompleteSelect(ev, ui);
+                return;
+            }
+            if (this.discussViewOwnerAsMobileAddItemHeader) {
+                this.discussViewOwnerAsMobileAddItemHeader.onMobileAddItemHeaderInputSelect(ev, ui);
+                return;
+            }
+            if (this.messagingMenuOwnerAsMobileNewMessageInput) {
+                this.messagingMenuOwnerAsMobileNewMessageInput.onMobileNewMessageInputSelect(ev, ui);
+                return;
+            }
+        },
+        /**
+         * @private
+         * @param {Object} req
+         * @param {function} res
+         */
+        onSource(req, res) {
+            if (this.chatWindowOwnerAsNewMessage) {
+                this.chatWindowOwnerAsNewMessage.onAutocompleteSource(req, res);
+            } else if (this.discussSidebarCategoryOwnerAsAddingItem) {
```

Comment:
Let's play safe and guard all `on` methods in diff here with `if (!this.exists()) { return; }`.
I'm preparing a PR to do this quick-and-dirty fix on all `on` methods in models.
In following days, will do more cleaning in models so that we won't have to explicitly put this guard as preamble of all handlers methods (as all methods will have this implicit guard).